packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -137,6 +136,20 @@ export class WorkspacesService extends ImmutableStore<WorkspacesState> {
       });
     };
 
+    // empty cluster URI - no cluster selected
+    if (!clusterUri) {
+      setWorkspace();
+      return Promise.resolve();
+    }
```

Comment:
It looks like you might have forgot to remove this bit, no?

Reply:
Yes, thanks